docs/cloud/cloud-authentication/cloud-auth-revoke-key.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Revoke Key
+layout: default
+parent: Manage API Keys
+grand_parent: Cloud
+nav_order: 5
+---
+
+# How do I revoke an API key in FeatureBase Cloud?
+{: .no_toc }
+
+Learn how to revoke a FeatureBase Cloud API key.
```

Comment:
You could add some further information to this which could be lifted from cloud-auth-manage.md

The content I'm talking about could be created as an "include" reusable file and could be used in both places.

For example:
* save the file to /_includes/cloud-auth/cloud-auth-key-revoke-warning.md
* content of file would be something like:

```
{: .warning}
>FeatureBase does not store API Key secrets.
>Organizations should revoke the API key if the secret is lost.
```
And this could then be inserted into each page by way of a shortcode:

```
{% include /cloud-auth/cloud-auth-key-revoke-warning.md %}
```

Reply:
I created an include and put it on the manage/revoke pages, but now the revoke page feels a little busy with a warning and then more warnings right below. Is this fine, or should this be rearranged in some capacity?